ovos_plugin_manager/templates/solvers.py

```diff
@@ -1,3 +1,31 @@
+# NEON AI (TM) SOFTWARE, Software Development Kit & Application Framework
```

Comment:
suggestion, add link to the original repo in here too https://github.com/NeonGeckoCom/neon_solvers

Reply:
Missed that this was committed; I had intended to put this in its own PR but that's a good idea to add

Reply:
https://github.com/OpenVoiceOS/ovos-plugin-manager/pull/185